packages/wallet-sdk/src/CoinbaseWalletProvider.test.ts

```diff
@@ -100,8 +107,9 @@ describe('signer configuration', () => {
 
   it('should throw error if signer is not initialized', async () => {
     const provider = createProvider();
-    await expect(provider.request({ method: 'personal_sign' })).rejects.toThrow(
-      `Must call 'eth_requestAccounts' before other methods`
-    );
+    await expect(provider.request({ method: 'personal_sign' })).rejects.toMatchObject({
+      code: standardErrorCodes.provider.unauthorized,
+      message: `Must call 'eth_requestAccounts' before other methods`,
+    });
```

Comment:
same as above.
the error should have been a proper error object containing message and a standard code.